.github/workflows/update-build-dashboard-data.yml

```diff
@@ -38,3 +38,4 @@ jobs:
             git add .
             git commit -m "Update build status data for dashboard"
             git push -f origin HEAD:refs/heads/documentation
+        continue-on-error: true
```

Comment:
This is not actually fixing the issue, right? Can we please update the commit and PR title?

Reply:
Thank you for the explanation.